parallel-install/README.md

```diff
@@ -2,23 +2,40 @@
 
 ## Overview
 
-The `parallel-install` package allows you to install and uninstall Kyma on the already existing clusters.
+The `parallel-install` library can install and uninstall Kyma on the already existing clusters.
+The library allows you to configure several parallel workers. This feature significantly reduces the time of the operation.
+Remember that parallel processing works only if the components (Helm Releases) are independent of each other.
 
 ## Usage
 
-Before starting the installation or uninstallation process, you need to provide a complete configuration by creating an instance of the `Installation` struct. To do so, use the `installation.NewInstallation` function.
+The top-level interface for library users is defined in the `installation` package, in the `Installer` interface.
+Before starting the installation or uninstallation process, you need to provide a complete configuration
+by creating an instance of the `Installation` struct.
+To do so, provide the `installation.NewInstallation` function with necessary parameters:
 
 | Parameter | Type | Example value | Description |
 | --- | --- | --- | --- |
 | prerequisites | `[][]string` | `{"cluster-essentials", "kyma-system"}, {"istio", "istio-system"},` | Array of the component's name and Namespace pairs. These components will be installed first, linearly, in a declared order. |
 | componentsYaml | `string` | - | Content of the [Installation CR](https://kyma-project.io/docs/#custom-resource-installation). Components will be extracted and installed in parallel. |
 | overridesYaml | `[]string` | `{ "foo: bar", "val: example" }` | List of Helm overrides. The latter the override, the higher is its priority. |
 | resourcesPath | `string` | `/go/src/github.com/kyma-project/kyma/resources` | Path to the Kyma resources. |
-| concurrency | `int` | `3` | Specifies how many components are installed simultaneously. |
+| cfg | `config.Config` | - | Specifies fine-grained configuration for the installation process. See below for explanation. |
```

Comment:
```suggestion
| cfg | `config.Config` | - | Specifies fine-grained configuration for the installation process. See the table with `config.Config` configuration options for details. |
```

Reply:
I reworded it just to avoid "above" and "below" phrases as per [guidelines](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology).